lcd/config_downloader.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import uuid
+import requests
+
+class ConfigDownloader:
+    def __init__(self, config_path, remote_url):
+        self.config_path = config_path
+        self.remote_url  = remote_url
+        self.api_key      = uuid.uuid4()
+
+    def run(self):
+        print "Setting up a new account"
```

Comment:
yes, sure. 

but I thought logging might not be good to use for CLI instructions and there is currently no logging output for the other methods `download()` and `save()`